### DISCUSSION TITLE
When running hashlink build it cant load "lime.hdll"

### DISCUSSION THREAD

**Discussion by gamemake-eng** [HaxeFlixel/flixel#2737] [Q&A]
When I build my game for hashlink and run it says "fatal error: unable to load "lime.hdll"" even though it's there. I am using lime 8.0.0, haxeflixel 1.5.1, and hashlink 1.12

**Comment by Geokureli** [comment]
I would look here for answers:
https://community.openfl.org/t/hashlink-target-build-problems/11447
